packages/mds-api-server/middleware/request-logging.ts

```diff
@@ -65,11 +65,9 @@ export const RequestLoggingMiddleware = ({
     return next()
   },
   (req: ApiRequest, res: ApiResponse, next: express.NextFunction) => {
-    const { REQUEST_DEBUG } = process.env
-
-    if (REQUEST_DEBUG === 'true' && !excludePaths.some(path => req.path.match(path))) {
+    if (!excludePaths.some(path => req.path.match(path))) {
       const { path, params, query, body } = req
-      logger.info('REQUEST_DEBUG', { path, params, query, body })
+      ApiServerLogger.debug('Request details', { path, params, query, body })
```

Comment:
I've observed that `params` hasn't been parsed yet at this point in the middleware pipeline. It seems to always be `{}` so it doesn't have much value. The data will be in the path though, so perhaps we should just remove.